### DISCUSSION TITLE
industry and sector

### DISCUSSION THREAD

**Discussion by christangga** [alvarobartt/investpy#305] [Q&A] [answered]
hi, first of all, I want to thank you for the great APIs that you've built.
I just want to ask a little thing, I went through the stocks API but didn't find industry and sector field. is there any API that we could hit to get the data?

**Comment by alvarobartt** [comment, marked as answer]
Hi @christangga, thank you for your kind words! 🤗 

There's an open issue to include it, https://github.com/alvarobartt/investpy/issues/273 so that I'll include those values in the `v1.0.4` release soon. Anyway, let me know if those are the values you'd like `investpy` to return. Thank you!
